_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,489 @@
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+The [instrument][] to toggle the navigational [block of repeated content][] is not [included in the accessibility tree][].
+
+```html
+<html>
+	<head>
+		<script src="../test-assets/bypass-blocks-cf77f2/toggle-display.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="#" aria-hidden="true" onclick="toggleHidden('chapters-navigation')">Toggle table of content</a>
+
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+There is no [instrument][] to toggle the [inclusion in the accessibility tree][included in the accessibility tree] of the navigational [block of repeated content][].
+
+```html
+<html>
+	<head>
+		<link rel="stylesheet" href="../test-assets/bypass-blocks-cf77f2/styles.css" />
+		<script src="../test-assets/bypass-blocks-cf77f2/toggle-display.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="#" onclick="toggleVisibility('chapters-navigation')">Toggle table of content</a>
+
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+There is no [instrument][] to toggle the [visibility][visible] of the navigational [block of repeated content][].
+
+```html
+<html>
+	<head>
+		<script src="../test-assets/bypass-blocks-cf77f2/toggle-display.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="#" onclick="toggleAriaHidden('chapters-navigation')">Toggle table of content</a>
+
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+### Inapplicable
```

Comment:
Passed example #7 can be used here?

Reply:
No, the Applicability is only about being an HTML document.
Block of repeated content is a subjective definition (because of the possibility for two blocks to be "equivalent resources" which is itself subject to interpretation). Therefore, it cannot be used in the Applicability. This in turn shaped the architecture of several rules (and notably the presence of the "document has no block of repeated content" rule.

The Expectations contain a "for each block of repeated content" quantification which is, by definition, true when there is no block of repeated content.

Reply:
So the only applicability for this rule is the HTML document, nevertheless, it has any block of repeated content or not. However, from the description, passed, and failed examples are "block of repeated content" objective.